### PR TITLE
Addling line height in configuration

### PR DESCRIPTION
**Pull request by alexookah**

### CODE REVIEW

**Comment by alexookah** [comment]
#78

File: AnimatedTextInput/Classes/AnimatedTextInput.swift [hunk 1]
```diff
@@ -242,7 +242,7 @@ open class AnimatedTextInput: UIControl {
         removeConstraints(constraints)
         pinLeading(toLeadingOf: lineView, constant: style.leftMargin)
         pinTrailing(toTrailingOf: lineView, constant: style.rightMargin)
-        lineView.setHeight(to: lineWidth)
+        lineView.setHeight(to: style.lineHeight)
```
Comment (victorBaro) [inline AnimatedTextInput/Classes/AnimatedTextInput.swift:245]: Remove `lineWidth` let declared on top of the file

File: AnimatedTextInput/Classes/AnimatedTextInputStyle.swift [hunk 2]
```diff
@@ -25,8 +26,9 @@ public struct AnimatedTextInputStyleBlue: AnimatedTextInputStyle {
     public let activeColor = UIColor(red: 51.0/255.0, green: 175.0/255.0, blue: 236.0/255.0, alpha: 1.0)
     public let placeholderInactiveColor = UIColor.gray.withAlphaComponent(0.5)
     public let inactiveColor = UIColor.gray.withAlphaComponent(0.5)
-    public let lineInactiveColor = UIColor.gray.withAlphaComponent(0.2)
     public let lineActiveColor = UIColor.gray.withAlphaComponent(0.0)
+    public let lineInactiveColor = UIColor.gray.withAlphaComponent(0.2)
+    public let lineHeight: CGFloat = 1.0
```
Comment (victorBaro) [inline AnimatedTextInput/Classes/AnimatedTextInputStyle.swift:31]: Please use: 
`public let lineHeight: CGFloat = 1.0 / UIScreen.main.scale` to maintain original value.

**Review by victorBaro — CHANGES_REQUESTED** [review]
Fix a couple of issues before being able to merge. Thank you for your PR!

**Comment by alexookah** [comment]
thanks for your feedback. somehow my repository with my latest pull request were removed. so thats why i made a new pull request with these changes check it

**Comment by alexookah** [comment]
see the new reuqest https://github.com/jobandtalent/AnimatedTextInput/pull/89

**Pull request closed without merging**